torch/_dynamo/variables/builtin.py

```diff
@@ -1572,7 +1572,7 @@ def call_hasattr(self, tx: "InstructionTranslator", obj, attr):
             name = attr.as_python_constant()
             if isinstance(obj, variables.BuiltinVariable):
                 return variables.ConstantVariable(hasattr(obj.fn, name))
-            return obj.call_hasattr(tx, name)
+            return obj.call_obj_hasattr(tx, name)
 
```

Comment:
Should we have `unimplemented` here? Replacing `call_hasattr` with `call_obj_hasattr` seems somewhat odd and specific to `hasattr`. If you show me the issue, I can try to debug.